<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmGaugeColor</class>
 <widget class="QWidget" name="frmGaugeColor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="GaugeColor" name="gaugeColor" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="cboxPointerStyle">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>圆形指示器</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>指针指示器</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>圆角指针指示器</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>三角形指示器</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ckAnimation">
       <property name="text">
        <string>动画</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GaugeColor</class>
   <extends>QWidget</extends>
   <header>gaugecolor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
